### DISCUSSION TITLE
Can I have a Blog with categories / archive enabled but disabled in nav?

### DISCUSSION THREAD

**Discussion by ironicbadger** [squidfunk/mkdocs-material#5863] [General]
As per title.

**Comment by squidfunk** [comment]
Currently not and not planned. You can raise a [change request](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/), if you like, explaining why you need it and how you'd imagine it to work. We need some more information to decide whether we can commit to maintain this functionality.
